lib/bundler/audit/database.rb

```diff
@@ -159,14 +162,52 @@ def advisories_for(name)
       # @return [Enumerator]
       #   If no block is given, an Enumerator will be returned.
       #
-      def check_gem(gem)
-        return enum_for(__method__,gem) unless block_given?
+      def check_gem(gem,&block)
+        check(gem,'gems',&block)
+      end
 
-        advisories_for(gem.name) do |advisory|
-          if advisory.vulnerable?(gem.version)
-            yield advisory
-          end
-        end
+      #
+      # Verifies whether the Ruby is effected by any advisories.
```

Comment:
Yeah, I'd opted for consistency over correctness (the existing documentation/specs all use "effect"). But in light of this feedback, I'll change them all. If you'd rather only see the new ones changed, just let me know.